staging/gatekeeper/values.yaml

```diff
@@ -19,19 +16,23 @@ commonLabels: {}
 # scmhash: abc123
 # myLabel: aakkmd
 
-image:
-  # image.name is the name of the Gatekeeper container image.
-  name: quay.io/open-policy-agent/gatekeeper
-  # image.tag is the tag name of the Gatekeeper container image.
-  tag:
-  # image.pullPolicy specifies when to pull the image.
-  pullPolicy: IfNotPresent
+## Number of replicas by default
+replicas: 1
```

Comment:
The default replicas are 3 in the chart repository 🤔 . I am wondering whether we need to deploy 2 at least.